### PR TITLE
feat: add release history

### PR DESCRIPTION
**Pull request by davidnuescheler**
https://release-history--helix-website--adobe.hlx.live/docs/release-history

vs.

https://main--helix-website--adobe.hlx.live/docs/release-history

### CODE REVIEW

File: blocks/releases/releases.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    .replace(/\[([^\]]+)\]\(([^)]+)\)/gim, '<a href="$2">$1</a>'); // links
+  return toHTML.trim(); // using trim method to remove whitespace
+}
+
+const displayNames = {
+  'helix-pipeline-service': 'Delivery Pipeline Service',
+  'helix-admin': 'Admin Service / API',
+  'helix-importer-ui': 'Content Importer Tooling',
+  'helix-cli': 'AEM Developer Command Line Interface',
+  'helix-sidekick-extension': 'AEM Authoring Sidekick',
```
Comment (rofe) [inline blocks/releases/releases.js:18]: I would remove product branding and simplify:
```suggestion
  'helix-pipeline-service': 'Delivery Pipeline',
  'helix-admin': 'Admin API',
  'helix-importer-ui': 'Content Importer',
  'helix-cli': 'Command Line Interface',
  'helix-sidekick-extension': 'Sidekick Extension',
```

File: blocks/releases/releases.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    cb.value = repo;
+    cb.id = repo;
+    control.append(cb);
+    const label = document.createElement('label');
+    label.textContent = displayNames[repo];
+    label.labelFor = repo;
+    control.append(label);
+    controls.append(control);
+    control.addEventListener('click', () => {
+      cb.checked = !cb.checked;
```
Comment (rofe) [inline blocks/releases/releases.js:90]: Should we push selected filters to the `hash` (if not 'all') to create sticky URLs?

File: blocks/releases/releases.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  return div;
+}
+
+export default async function decorate(block) {
+  const url = block.querySelector('a').href;
+  block.textContent = '';
+  const releases = document.createElement('div');
+  const controls = document.createElement('div');
+  controls.classList.add('releases-controls');
+  Object.keys(displayNames).forEach((repo) => {
```
Comment (rofe) [inline blocks/releases/releases.js:74]: I would add an 'all' option at the beginning

**Review by rofe — APPROVED** [review]
Very cool! Nothing critical in my review, just some suggestions.

PS: I like the colored release boxes. They look a bit like cards on mobile - should we turn them into a card grid on desktop?

**Comment by davidnuescheler** [comment]
@rofe i merged it... happy to look at more items later. personally i think that one of the best features would be the `#` link we have on the rest of the doc, feel free to that...

**Pull request merged**